devel/clean

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Cleans up after the build process by removing build artifacts and caches.
+#
+set -euo pipefail
+
+repo="$(cd "$(dirname "$0")/.."; pwd)"
+registry=nextstrain-local-registry
+build_dir="$repo"/build
+builder=nextstrain-builder
+
+
+echo "--> Deleting $registry container"
+
+if docker container inspect "$registry" &>/dev/null; then
+    "$repo"/devel/stop-localhost-registry
+else
+    echo "skipped; $registry container does not exist"
+fi
+
+
+echo "--> Deleting artifacts in $build_dir"
+
+if [[ -d "$build_dir" ]]; then
+    du -hs "$build_dir"
+    rm -rf "$build_dir"
+else
+    echo "skipped; $build_dir does not exist"
+fi
+
+
+echo "--> Deleting $builder (and its caches)"
+
+if docker buildx inspect "$builder" &>/dev/null; then
+    docker buildx du --builder "$builder"
+    docker buildx rm --builder "$builder"
+else
+    echo "skipped; $builder does not exist"
+fi
+
+
+echo "--> Deleting local images"
```

Comment:
```suggestion
echo "--> Deleting local images pulled from localhost:5000"
```

Clarify that this does not delete local images pulled from elsewhere (e.g. Docker Hub).

Reply:
Done with repush in 23a8431.